code/modules/mob/living/simple_animal/abnormality/waw/apex_predator.dm

```diff
@@ -0,0 +1,188 @@
+/mob/living/simple_animal/hostile/abnormality/apex_predator
+	name = "Apex Predator"
+	desc = "An abnormality resembling a beaten up crash dummy."
+	icon = 'ModularTegustation/Teguicons/64x64.dmi'
+	icon_state = "apex"
+	icon_living = "apex"
+	pixel_x = -16
+	base_pixel_x = -16
+
+
+	maxHealth = 1600
+	health = 1600
+	damage_coeff = list(BRUTE = 1, RED_DAMAGE = 0.2, WHITE_DAMAGE = 1, BLACK_DAMAGE = 1, PALE_DAMAGE = 1)
+	ranged = TRUE
+	melee_damage_lower = 30
+	melee_damage_upper = 40
+	move_to_delay = 3
+
+	melee_damage_type = RED_DAMAGE
+	armortype = RED_DAMAGE
+	stat_attack = HARD_CRIT
+
+	fear_level = 0	//You should never notice it
+
+	attack_sound = 'sound/abnormalities/fragment/attack.ogg'
+	attack_verb_continuous = "stabs"
+	attack_verb_simple = "stab"
+	faction = list("hostile")
+	can_breach = TRUE
+	threat_level = WAW_LEVEL
+	start_qliphoth = 2
+	work_chances = list(
+						ABNORMALITY_WORK_INSTINCT = list(0, 0, 30, 35, 40),
+						ABNORMALITY_WORK_INSIGHT = list(0, 0, 30, 35, 40),
+						ABNORMALITY_WORK_ATTACHMENT = list(25, 20, 15, 10, 0),
+						ABNORMALITY_WORK_REPRESSION = list(0, 0, 50, 55, 55)
+						)
+	work_damage_amount = 10
+	work_damage_type = RED_DAMAGE
+
+	ego_list = list(
+		/datum/ego_datum/weapon/animalism,
+		/datum/ego_datum/armor/animalism
+		)
+//	gift_type =  /datum/ego_gifts/animalism
+
+	var/revealed = TRUE
+	var/can_act = TRUE
+	var/backstab_damage = 200
+	var/agent_status //Used for insanity
+
+	var/jumping	//Used so it can only start one jump at once
+	var/busy	//Can we move now?
+
+	var/jump_cooldown
+	var/jump_cooldown_time = 5 SECONDS
+	var/jump_damage = 60 // Scales with distance; Ideally, you shouldn't be able to outheal it with white V armor or less
```

Comment:
Seeing as this is Red Damage I'm assuming Blue Star copypasta.

Reply:
Just for cooldown, easier to copypaste than write from scratch. I swore I replaced the comment though.